src/ngraph/validation_util.hpp

```diff
@@ -114,6 +114,32 @@ namespace ngraph
     ///            by adding tensor_rank to axis.
     int64_t normalize_axis(const Node* node, std::int64_t axis, std::int64_t tensor_rank);
 
+    /// \brief      Handle out of range axes in vector.
+    ///
+    /// \param[in]  node_description  The name of node with requested axes.
+    /// \param[in]  axes              The requested vector of axes.
+    /// \param[in]  tensor_rank       The corresponding tensor rank.
+    ///
+    /// \return     If any negative axis in vector, it counts from the last to the first
+    ///             axis, by adding tensor_rank to axis.
+    ///
+    std::vector<size_t> normalize_axes(const std::string& node_description,
```

Comment:
The fact that we have here five functions named (almost exactly) the same confuses me a lot. This also gives me a sign that we doesn't have here a clear API. 

Only ONNX Importer uses this overload with vector parameter so IMHO this should be ONNX-specific.

Reply:
I see your point but purpose of the ask is having one place for all `normalize` methods